Gaming The Syllabus/Assets/Scripts/Combat/UnitAction.cs

```diff
@@ -67,6 +79,10 @@ public XmlSchema GetSchema()
 
     public void ReadXml(XmlReader reader)
```

Comment:
Dang, awesome work here. This XmlReader stuff is a nightmare and you figured it out really well